schema.graphql

```diff
@@ -237,3 +237,25 @@ type Solution @entity {
   txHash: Bytes!
   txLogIndex: BigInt!
 }
+
+type Stats @entity {
+  id: ID!
+
+  # The total volume denominated in OWL (all sell amounts combined)
```

Comment:
Love the comments! I think GraphQL has support for "doc-comments" so that the schema explorer thingy in the the Graph explorer can show this documentation.

I think it can be either:
```suggestion
  "The total volume denominated in OWL (all sell amounts combined)"
```
or
```suggestion
  """
  The total volume denominated in OWL (all sell amounts combined)
  """
```

Reply:
FOund it! http://spec.graphql.org/June2018/#Description